modules/server/src/main/scala/netty.scala

```diff
@@ -45,10 +45,13 @@ object netty {
   final case class PermitKeepAliveTime(keepAliveTime: Long, timeUnit: TimeUnit) extends GrpcConfig
   final case class PermitKeepAliveWithoutCalls(permit: Boolean)                 extends GrpcConfig
 
-  final case class NettyServerConfigBuilder(initConfig: ChannelFor, configList: List[GrpcConfig]) {
+  // ?
```

Comment:
Not sure what this `?` means

Reply:
I didn't really like these case class builders, and wanted to see if I could clean those up a little bit as well, but I forgot about them.

Added a new commit moving / inlining the code to helper methods in `GrpcServer`.